bin/docker-start

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+cd /home/app/avalon
+
+# TODO: Avalon image doesn't have nc installed. The bundle install takes enough time
```

Comment:
Two comments (no ideas how much access/ability to change the dockerfile so your mileage may vary here):

- `nc` is a separate package. For example we are using an alpine based image in jupiter, so we install this here:
https://github.com/ualbertalib/jupiter/blob/master/Dockerfile#L7

  So could easily do the same for your avalon image if its debian based etc.

- If you have the power to change the image, bundle install should be done within the dockerfile, before syncing the files across. This is so the gems are cached into the image and anyone pulling this down doesn't have to do this. Having it here, means anytime the container is spun up, it runs bundle install (first time will be slow obviously, second time, won't be a huge issue) ¯\_(ツ)_/¯

Otherwise looks good :+1:

Reply:
Been looking into it -- it might be best if I start maintaining our own avalon image in Dockerhub. One strange issue with upstream's image is that on the dev branch they install the bundle without `development` / `test`  ¯\_(ツ)_/¯:

https://github.com/avalonmediasystem/avalon-docker/blob/6.x-dev/avalon/Dockerfile#L21

Reply:
Yeah that's a bit weird, seems to be their dockerfile is more geared toward production then for using with development/test (assets precompile/nginx/whenever crons/etc). 

Up to you, these were more suggestions then anything. If this works for now, then leave it

Reply:
Cheers, I'll explore this further outside of this PR. (Might also be worth spinning up a separate Resque container, like you guys do with Sidekiq).